Compiler Fix release build

From https://github.com/Samsung/ONE/issues/457

```
07:08:13 /home/jenkins_slave/workspace/nnfw/master/push-nncc-release/compiler/mir/src/mir_caffe_importer/caffe_op_creator.cpp:431:17: error: 'result' may be used uninitialized in this function [-Werror=maybe-uninitialized]
07:08:13    return {result};
07:08:13                  ^
07:08:13 cc1plus: all warnings being treated as errors
```
```
07:08:14 /home/jenkins_slave/workspace/nnfw/master/push-nncc-release/compiler/mir/src/mir_caffe2_importer/caffe2_op_creator.cpp:514:9: error: unused variable 'min' [-Werror=unused-variable]
07:08:14    float min = getSingleArgument(op, "min", float(0));
07:08:14          ^~~
```

--- a/compiler/mir/src/mir_caffe2_importer/caffe2_op_creator.cpp
+++ b/compiler/mir/src/mir_caffe2_importer/caffe2_op_creator.cpp
@@ -40,6 +40,7 @@
 #include "mir/TensorUtil.h"
 
 #include <cmath>
+#include <stdexcept>
 #include <vector>
 
 namespace mir_caffe2
@@ -513,7 +514,10 @@ Caffe2OpCreator::convertClip(const std::vector<mir::Operation::Output *> &inputs
   float max = getSingleArgument(op, "max", float(0));
   float min = getSingleArgument(op, "min", float(0));
 
-  assert(max > 0.0 && min == 0.0 && "Support only if clip is CappedRelu");
+  if (min != 0.0f)
+    throw std::runtime_error("Clip: min != 0 is not supported.");
+  if (max <= min)
+    throw std::runtime_error("Clip: max <= min is not supported.");
   auto cap_relu = createOp<ops::CappedReluOp>(inputs[0], max);
 
   return {cap_relu->getOutput(0)};

--- a/compiler/mir/src/mir_caffe_importer/caffe_op_creator.cpp
+++ b/compiler/mir/src/mir_caffe_importer/caffe_op_creator.cpp
@@ -43,6 +43,7 @@
 #include <cmath>
 #include <iostream>
 #include <set>
+#include <stdexcept>
 
 namespace mir_caffe
 {
@@ -425,7 +426,7 @@ CaffeOpCreator::convertPooling(const caffe::LayerParameter &layer,
       break;
     }
     default:
-      assert(false);
+      throw std::runtime_error("Unsupported PoolMethod: " + std::to_string(params.pool()));
   }
 
   return {result};

--- a/compiler/mir/src/mir_tflite_importer/tflite_op_creator.cpp
+++ b/compiler/mir/src/mir_tflite_importer/tflite_op_creator.cpp
@@ -50,6 +50,8 @@
 #include "mir/ShapeRange.h"
 #include "mir/Tensor.h"
 
+#include <stdexcept>
+
 namespace mir_tflite
 {
 
@@ -65,7 +67,8 @@ static mir::ops::PaddingType convertPadding(tflite::Padding padding)
     case tflite::Padding_SAME:
       return mir::ops::PaddingType::SameUpper;
     default:
-      assert(false);
+      throw std::runtime_error(std::string("Unsupported Padding: ") +
+                               tflite::EnumNamePadding(padding));
   }
 }
 

